R/zchunk_L202.an_input.R

```diff
@@ -0,0 +1,646 @@
+      add_title("Generic animal production subsector info") %>%
+      add_units("NA") %>%
+      add_comments("A_an_subsector written to all regions") %>%
+      add_legacy_name("L202.SubsectorAll_an") %>%
+      add_precursors("aglu/A_an_subsector", "common/GCAM_region_names") ->
+      L202.SubsectorAll_an
+
+    L202.StubTech_an %>%
+      add_title("Identification of stub technologies for animal production") %>%
+      add_units("NA") %>%
+      add_comments("A_an_technology written to all regions") %>%
+      add_legacy_name("L202.StubTech_an") %>%
+      add_precursors("aglu/A_an_technology", "common/GCAM_region_names") ->
+      L202.StubTech_an
+
+    L202.StubTechInterp_an %>%
+      add_title("Shareweight interpolation for animal production technologies") %>%
+      add_units("NA") %>%
+      add_comments("A_an_technology written to all regions") %>%
+      add_legacy_name("L202.StubTechInterp_an") %>%
+      add_precursors("aglu/A_an_technology", "common/GCAM_region_names") ->
+      L202.StubTechInterp_an
+
+    L202.StubTechProd_an %>%
+      add_title("Animal production by technology and region") %>%
+      add_units("Unitless") %>%
+      add_comments("Animal production written across model base years and regions") %>%
+      add_legacy_name("L202.StubTechProd_an") %>%
+      add_precursors("aglu/A_an_technology", "L107.an_Prod_Mt_R_C_Sys_Fd_Y", "common/GCAM_region_names") ->
+      L202.StubTechProd_an
+
+    L202.StubTechCoef_an %>%
+      add_title("Animal production input-output coefficients by technology and region") %>%
+      add_units("Unitless") %>%
+      add_comments("Animal production input-output coefficients written across model base years and regions") %>%
+      add_legacy_name("L202.StubTechCoef_an") %>%
+      add_precursors("aglu/A_an_technology", "L107.an_Prod_Mt_R_C_Sys_Fd_Y", "common/GCAM_region_names") ->
+      L202.StubTechCoef_an
+
+    L202.GlobalTechCost_an %>%
+      add_title("Costs of animal production technologies") %>%
+      add_units("billion USD") %>%
+      add_comments("Animal feed cost, prices, and technology") %>%
+      add_legacy_name("L202.GlobalTechCost_an") %>%
+      same_precursors_as(L202.StubTechCoef_an) %>%
+      add_precursors("L132.ag_an_For_Prices", "L107.an_Feed_Mt_R_C_Sys_Fd_Y") ->
+      L202.GlobalTechCost_an
+
+    L202.GlobalRenewTech_imp_an %>%
+      add_title("Generic technology info for animal imports") %>%
+      add_units("NA") %>%
+      add_comments("A_an_supplysector written to all model years and regions") %>%
+      add_legacy_name("L202.GlobalRenewTech_imp_an") %>%
+      add_precursors("aglu/A_an_supplysector", "common/GCAM_region_names") ->
+      L202.GlobalRenewTech_imp_an
+
+    L202.StubTechFixOut_imp_an %>%
+      add_title("Animal imports for net importing regions in all periods") %>%
+      add_units("Mt") %>%
+      add_comments("A_an_supplysector and animal product mass balances written to all model years and regions") %>%
```

Comment:
Prescribed trajectory of animal commodity imports by region; the final base year's values are copied forward to all future model time periods (imports and exports of these secondary commodities are not represented from economic competition).